File: app/src/main/java/com/artmurka/artmurkaapp/Model/Pojo/ItemList/ItemBasket/Success.java

package com.artmurka.artmurkaapp.Model.Pojo.ItemList.ItemBasket;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Success {

    @SerializedName("msg")
    @Expose
    private String msg;
    @SerializedName("basket")
    @Expose
    private Basket basket;

    public String getMsg() {
        return msg;
    }

    public void setMsg(String msg) {
        this.msg = msg;
    }

    public Basket getBasket() {
        return basket;
    }

    public void setBasket(Basket basket) {
        this.basket = basket;
    }

}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/itembasket/Success.kt
package com.artmurka.artmurkaapp.data.model.pojo.itemlist.itembasket

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Success {

    @SerializedName("msg")
    @Expose
    var msg: String? = null
    @SerializedName("basket")
    @Expose
    var basket: Basket? = null

}
